src/extension/app-schema/app-schema-test/src/test/java/org/geoserver/test/AbstractAppSchemaMockData.java

```diff
@@ -773,4 +841,67 @@ private String modifyOnlineMappingFileContent(String mappingFileName) throws IOE
             return content.toString();
         }
     }
+
+    private void updateURLStringInFiles() throws IOException {
+        String oldIncludedTypes =
+                IOUtils.toString(
+                        getClass()
+                                .getClassLoader()
+                                .getResourceAsStream(
+                                        "appschema/appschema-gml3132/includedTypes.xml"));
+        String newIncludedTypes =
+                oldIncludedTypes.replace("PATH_TO_BE_REPLACED", GEOPKG_DIR).trim();
+        try (ByteArrayInputStream bais =
+                new ByteArrayInputStream(newIncludedTypes.getBytes(StandardCharsets.UTF_8))) {
+            copy(bais, "appschema/appschema-gml3132/includedTypes.xml");
+        }
+
+        try (ByteArrayInputStream bais =
+                new ByteArrayInputStream(newIncludedTypes.getBytes(StandardCharsets.UTF_8))) {
+            copyTarget(bais, "appschema/appschema-gml3132/includedTypes.xml");
+        }
+
+        String oldMeteo =
+                IOUtils.toString(
+                        getClass()
+                                .getClassLoader()
+                                .getResourceAsStream(
+                                        "appschema/appschema-gml3132/meteo.appschema"));
+        String newMeteo = oldMeteo.replace("PATH_TO_BE_REPLACED", GEOPKG_DIR).trim();
+        try (ByteArrayInputStream bais =
+                new ByteArrayInputStream(newMeteo.getBytes(StandardCharsets.UTF_8))) {
+            copy(bais, "appschema/appschema-gml3132/meteo.appschema");
+        }
+
+        try (ByteArrayInputStream bais =
+                new ByteArrayInputStream(newIncludedTypes.getBytes(StandardCharsets.UTF_8))) {
+            copyTarget(bais, "appschema/appschema-gml3132/meteo.appschema");
+        }
+    }
+
+    private void copyGeopkgResourcesFolder() {
+        copy(
+                getClass().getClassLoader().getResourceAsStream("appschema/stations.gpkg"),
+                "appschema/stations.gpkg");
+        copy(
```

Comment:
why in addition to the geopackage you need to copy also the xsd and mapping files? For what I know mappings and xsd are already handled by the addFeatureType method. Did I miss something?
In case they are not needed please amend the code and remove the additional files in the resource folder.

Reply:
you are right. fixed.